module/helpers/utils.mjs

```diff
@@ -251,15 +251,15 @@ export const adjustRange = (rangeVal, decrease) => {
     return range[rangeKeys[newIndex]];
 };
 
-export const updateActorTokens = async (actor, update) => {
+export const updateActorTokens = async (actor, update, updateToken) => {
```

Comment:
Would it be possible to add a comment to this function? It was actually a bit hard for me to parse on review what this was doing. I think update is the prototype token update, and updateToken is the per token one?

Reply:
That's fair! Added.
```
/**
 * 
 * @param {DhActor} actor - The actor for which all tokens will run a data update.
 * @param {string} update - The data update to be applied to all tokens.
 * @param {func} updateToken - Optional, specific data update for the non-prototype tokens as a function using the token data. Useful to handle wildcard images where each token has a different image but the prototype has a wildcard path.
 */
```